src/components/Header.js

```diff
@@ -1,10 +1,61 @@
-import React from 'react';
-import styled from 'styled-components';
+import React from "react";
+import styled from "styled-components";
+import { images } from "./images/image";
 
-export default function Header() {
-	return <Wrapper>발신자 변경해주는 상단바</Wrapper>;
+export default function Header(props) {
+  //change parent's user state using props
+  const handleClick = () => {
+    props.onClick();
+  };
+
+  return (
+    <Profile onClick={handleClick}>
+      {/* if user value is false */}
+      {/* I hate this code!!!!!! I want simpler code... */}
+      {!props.user && (
+        <>
+          <img src={images.COOL} />
+          <div>
+            <h2 style={{ marginBottom: "0" }}>정쿨</h2>
+            <br />
+            현재 활동중
+          </div>
+        </>
+      )}
+      {/* if user value is true */}
+      {props.user && (
+        <>
+          <img src={images.EUNKO} />
+          <div>
+            <h2 style={{ marginBottom: "0" }}>고은</h2>
+            <br />
+            현재 활동중
+          </div>
+        </>
```

Comment:
props를 받아서 반복되는 컴포넌트를 이렇게 바꿀 수 있답니다!

아래 `div`, `h2 `태그도 styled-component연습을위해, 이걸 이용해서 스타일링해주시면 좋을 것 같아요ㅠㅠ
`h2`를 `UserName`과 같은 네이밍의 컴포넌트로,
현재활동중을 `UserOn`과 같은 네이밍의 컴포넌트로해서 만들어주시면 좋을 것 같습니다 :)

```suggestion
  <>
		<Img src={imgUrl} />
			<div>
			<h2 style={{ marginBottom: '0' }}>{user}</h2>
			<br />
			현재 활동중
			</div>
</>
```

Reply:
와... 안그래도 이 부분 짜면서 코드 반복이 심해서 해결 방법을 찾고 있었어요 ㅠㅠ
props를 이용하는 방법이 있었군요!! 감사합니다 ㅠㅠ